### PR TITLE
Update japanese translation.

### PR DESCRIPTION
**Pull request by niyari**
## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [x] Internationalization and localization
- [ ] Other (please describe):

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?

- Update japanese translation.
-
-

## Quality check

Before creating this PR, have you:

- [x] Followed the code style guideline as described in [CONTRIBUTING.md](https://github.com/veler/DevToys/blob/main/CONTRIBUTING.md)
- [x] Verified that the change work in Release build configuration
- [x] Checked all unit tests pass

### CODE REVIEW

**Comment by veler** [comment]
Thank you very much! :)

**Comment by inductor** [comment]
@veler Would you mind adding me as a reviewer next time just in case, for localization? not sure if it's too much work for you but that'd be much appreciated as who started localizing this to Japanese :)

File: src/dev/impl/DevToys/Strings/ja-JP/GZipEncoderDecoder.resw [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+    <value>GZip 圧縮展開ツール</value>
+  </data>
+  <data name="Ascii" xml:space="preserve">
+    <value>ASCII</value>
+  </data>
+  <data name="ConfigurationTitle" xml:space="preserve">
+    <value>設定</value>
+  </data>
+  <data name="ConversionDecompress" xml:space="preserve">
+    <value>展開</value>
```
Comment (inductor) [inline src/dev/impl/DevToys/Strings/ja-JP/GZipEncoderDecoder.resw:130]: @niyari This is just my opinion but `解凍` sounds more common for an antonym of `圧縮`, what do you think?

**Comment by veler** [comment]
> @veler Would you mind adding me as a reviewer next time just in case, for localization? not sure if it's too much work for you but that'd be much appreciated as who started localizing this to Japanese :)

Absolutely! I can make you our main maintainer of Japanese language. Thank you very much for asking that, it's actually a great thing! :D

File: src/dev/impl/DevToys/Strings/ja-JP/GZipEncoderDecoder.resw (hunk 1 above)
Comment (niyari) [reply src/dev/impl/DevToys/Strings/ja-JP/GZipEncoderDecoder.resw:130]: "解凍" is a term that has been used for more than 30 years, But keeping it the same as OS localization is also a kindness to users.
image
(Interestingly, "解凍" is also used in the result of this command.)

We also use the term "伸張", but it is not common, and we adopted "展開" as the word that best fits the current feature.

**Pull request merged**